### PR TITLE
[CloudMinds] 10326-Robot PM

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes all operations on CCUs.

### CODE REVIEW

File: 10326.xml [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+      </Item>
+
+      <Item ID="300">
+        <Name>Battery Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 2]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[Battery Temperature.]]></Description>
+      </Item>	  
+	  <Item ID="301">
+        <Name>Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 3]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[PM2.5 of the environment.]]></Description>
+      </Item>
+      <Item ID="304">
+        <Name>Smog</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 4]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[Environment Humidity.]]></Description>
+      </Item>
+      <Item ID="303">
+        <Name>PM2.5</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 5]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[Smoke concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="305">
+        <Name>CO</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
```
Comment (seanmcilroy29) [inline 10326.xml:113]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 6]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[CO concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="306">
+        <Name>CO2</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
```
Comment (seanmcilroy29) [inline 10326.xml:123]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 7]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[CO2 concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="307">
+        <Name>PM10</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 8]
```diff
@@ -0,0 +1,724 @@
...
+	  
+	  
+      <Item ID="601">
+        <Name>Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>M/H</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 9]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[The percentage of Dust Box Space used for CloudCleanX.]]></Description>
+      </Item>
+	  <Item ID="604">
+        <Name>Obstacle Distance</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>cm</Units>
```
Comment (seanmcilroy29) [inline 10326.xml:173]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 10]
```diff
@@ -0,0 +1,724 @@
...
+      </Item>
+	  
+	  <Item ID="607">
+        <Name>Robot Temperate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 11]
```diff
@@ -0,0 +1,724 @@
...
+      </Item>	  
+	  
+	  <Item ID="1101">
+        <Name>Data Traffic Used</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml [hunk 12]
```diff
@@ -0,0 +1,724 @@
...
+        <Description><![CDATA[How many questions can't be answered by the robot.]]></Description>
+      </Item>
+	  <Item ID="1113">
+        <Name>Mileage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
```
Comment (seanmcilroy29) [inline 10326.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10326.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10326.xml]: Have commented in another object.

File: 10326.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10326.xml]: Have commented in another object.

File: 10326.xml (hunk 5 above)
Comment (Kenny-SZ) [reply 10326.xml:113]: ppm, parts per million, the unit of gas concentration.

File: 10326.xml (hunk 6 above)
Comment (Kenny-SZ) [reply 10326.xml:123]: ppm, parts per million, the unit of gas concentration.

File: 10326.xml (hunk 7 above)
Comment (Kenny-SZ) [reply 10326.xml]: Have commented.

File: 10326.xml (hunk 8 above)
Comment (Kenny-SZ) [reply 10326.xml]: Have commented

File: 10326.xml (hunk 10 above)
Comment (Kenny-SZ) [reply 10326.xml]: Have commented before.

File: 10326.xml (hunk 9 above)
Comment (Kenny-SZ) [reply 10326.xml:173]: cm(centimeter), the unit of distance.

File: 10326.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10326.xml]: μg/m3 is the unit of PM2.5, the concentration of an air pollutant is given in micrograms (one-millionth of a gram) per cubic meter air.

File: 10326.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10326.xml]: μg/m3 is the unit of PM2.5, the concentration of an air pollutant is given in micrograms (one-millionth of a gram) per cubic meter air.

File: 10326.xml (hunk 11 above)
Comment (Kenny-SZ) [reply 10326.xml]: Mbps, the unit of data traffic, million bits per second.

File: 10326.xml (hunk 12 above)
Comment (Kenny-SZ) [reply 10326.xml]: M（meter）, the unit of distance.

Comment (seanmcilroy29) [reply 10326.xml]: ``M`` should be replaced with ``m``, as per SenML

Comment (Kenny-SZ) [reply 10326.xml]: Changed(M to m, M/H to m/h), thanks.

File: 10326.xml (hunk 3 above)
Comment (seanmcilroy29) [reply 10326.xml]: replace μg/m3 with ug/m3 which is micrograms per cubic meter

File: 10326.xml (hunk 4 above)
Comment (seanmcilroy29) [reply 10326.xml]: replace μg/m3 with ug/m3 which is micrograms per cubic meter

File: 10326.xml (hunk 7 above)
Comment (seanmcilroy29) [reply 10326.xml]: replace μg/m3 with ug/m3 which is micrograms per cubic meter

Comment (Kenny-SZ) [reply 10326.xml]: Replaced.

File: 10326.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10326.xml]: Replaced, thanks.

File: 10326.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10326.xml]: Replaced, thanks.

**Comment by Kenny-SZ** [comment]
Changed % to /100

**Comment by Kenny-SZ** [comment]
Changed Mbps to Mbit/s.

**Pull request merged**